pkg/platform/model/vcs.go

```diff
@@ -64,8 +64,11 @@ const (
 	// NamespaceLanguageMatch is the namespace used for language requirements
 	NamespaceLanguageMatch = `^language$`
 
-	// NamespacePackageMatch is the namespace used for package requirements
-	NamespacePackageMatch = `^language\/\w+$`
+	// NamespaceLanguagePackageMatch is the namespace used for language package requirements
+	NamespaceLanguagePackageMatch = `^language\/\w+$`
+
+	// NamespaceBundlesPackageMatch is the namespace used for bundle package requirements
+	NamespaceBundlesPackageMatch = `^bundles\/\w+$`
```

Comment:
This should be `NamespaceBundlesMatch`. Bundles aren't packages.

Reply:
That's good to know. Not obvious since `state packages` is now handling both and neither of the namespaces have any mention of `package`